### PR TITLE
[zh-cn]: Update translation of Number.constructor

### PR DESCRIPTION
**Pull request by JasonLamv-t**
### Description

Same as the title.

### Motivation

Ditto.

### Additional details

None.

### Related issues and pull requests

#15736

### CODE REVIEW

File: files/zh-cn/web/javascript/reference/global_objects/number/number/index.md [hunk 1]
```diff
@@ -23,11 +23,11 @@ Number(value)
 
 ### 返回值
 
-当 `Number` 作为构造函数调用（使用 [`new`](/zh-CN/docs/Web/JavaScript/Reference/Operators/new)）时，它创建一个 {{jsxref("Number")}} 对象，并不是原始值。
+当 `Number` 作为构造函数调用（使用 [`new`](/zh-CN/docs/Web/JavaScript/Reference/Operators/new)）时，它创建一个 {{jsxref("Number")}} 对象，这个对象**不是**原始值。
 
-当 `Number` 作为普通函数调用时，它[将参数强制转换为数值原始值](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Number#数值强制转换)。[BigInt](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/BigInt) 被转换为数值。如果值不能转换，则返回 {{jsxref("NaN")}}。
+当 `Number` 作为普通函数调用时，它[将参数强制转换为数字原始值](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Number#Number_强制转换)。[BigInt](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/BigInt) 被转换为数字。如果值不能转换，则返回 {{jsxref("NaN")}}。
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/number/number/index.md:28]:
```suggestion
当 `Number` 作为普通函数调用时，它[将参数强制转换为数字原始值](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Number#number_强制转换)。[BigInt](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/BigInt) 被转换为数字。如果值不能转换，则返回 {{jsxref("NaN")}}。
```

File: files/zh-cn/web/javascript/reference/global_objects/number/number/index.md [hunk 2]
```diff
@@ -42,9 +42,9 @@ typeof a; // "object"
 typeof b; // "number"
 ```
 
-### 使用 Number() 将 BigInt 转换为数值
+### 使用 Number() 将 BigInt 转换为数字
 
-`Number()` 是唯一可以将 BigInt 转换为数字而不抛出（错误）的情况，因为它非常明确。
+`Number()` 是唯一可以将 BigInt 转换为数字而不抛出错误的情况，因为它非常明确。
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/number/number/index.md:47]: 感觉这里的“explicit”翻译为“明确”怪怪的，应该还是“显式”的意思，考虑一下和“very”放在一起应该怎么翻译

Comment (JasonLamv-t) [reply files/zh-cn/web/javascript/reference/global_objects/number/number/index.md:47]:
```suggestion
`Number()` 是唯一可以将 BigInt 转换为数字而不抛出错误的情况，因为其预期行为非常明确。
```

**Review by yin1999 — APPROVED** [review]
Thank you @JasonLamv-t. Sorry for the late as I'm a bit busy this weekend.

File: files/zh-cn/web/javascript/reference/global_objects/number/number/index.md [hunk 3]
```diff
@@ -42,9 +42,9 @@ typeof a; // "object"
 typeof b; // "number"
 ```
 
-### 使用 Number() 将 BigInt 转换为数值
+### 使用 Number() 将 BigInt 转换为数字
 
-`Number()` 是唯一可以将 BigInt 转换为数字而不抛出（错误）的情况，因为它非常明确。
+`Number()` 是唯一可以将 BigInt 转换为数字而不抛出错误的情况，因为其预期行为非常明确。
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/number/number/index.md:47]:
```suggestion
`Number()` 是唯一可以将 BigInt 转换为数字而不抛出错误的情况，因为这是完全显式的转换。
```

**Review by yin1999 — COMMENTED** [review]
Emmmm. Just a suggestion regarding what was previously discussed.

**Pull request merged**